config/locales/emails.fr.yml

```diff
@@ -0,0 +1,60 @@
+fr:
+  emails:
+    event_acceptance:
+      subject: "Votre demande pour %{conference}"
+      greeting: "Cher %{forename},"
```

Comment:
"Cher·e" (or something like that? /me is always confused with this types of words)